customers.sql

```diff
@@ -0,0 +1,8 @@
+select 
+o.customer_id, c.name, c.email, min(o.created_at) as first_order_at, count(o.id) as number_of_orders
+from `analytics-engineers-club.coffee_shop.customers` c
+join `analytics-engineers-club.coffee_shop.orders` o
+  on c.id = o.customer_id
+group by o.customer_id, c.name, c.email
+order by first_order_at
+limit 5;
```

Comment:
Specify ORDER BY desc or asc

Reply:
Overall I'd write the query like this 

```
select 
  orders.customer_id, 
  customers.name, 
  customers.email, 
  min(orders.created_at) as first_order_at, 
  count(orders.id) as number_of_orders

from `analytics-engineers-club.coffee_shop.customers` customers
left join `analytics-engineers-club.coffee_shop.orders` orders
  on customers.id = orders.customer_id
group by 1,2,3
order by first_order_at asc
limit 5;
```

Reply:
similar to my preference re qualifying joins, I try to avoid qualifying default behavior.